app/src/pages/common/membersPage/modals/inviteUserModal/inviteUserModal.jsx

```diff
@@ -236,7 +248,7 @@ export class InviteUserModal extends Component {
       const foundUser = foundUsers?.content.find(({ email }) => email === data.email);
       if (foundUser) {
         userData.user = makeOptions(data.project, false)({ content: [foundUser] })[0];
-      } else {
+      } else if (!ssoUsersOnly) {
```

Comment:
Do we need this check here?
Looks like if the external user invites disabled on the `InputUserSearch` level, this block will not be executed in any case.

Reply:
Absolutely